src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/interaction.cpp

```diff
@@ -171,8 +171,20 @@ TEST_P(IntertactionCPUTest, CompareWithRefs) {
     CheckNumberOfNodesWithType(compiledModel, "Interaction", 1);
 }
 
+using IntertactionCPUTest_FP16 = IntertactionCPUTest;
+TEST_P(IntertactionCPUTest_FP16, CompareWithRefs) {
+    if (!(ov::with_cpu_x86_avx512_core_fp16() || ov::with_cpu_x86_avx512_core_amx_fp16())) {
```

Comment:
Here and in other such places.
Isn't checking for `with_cpu_x86_avx512_core_fp16()` is just enough? I can't recall any machines with AMX_FP16 support but without AVX512_FP16 extension.

Reply:
already updated it